pipedrive/example-apps
# GRAM-2947 Clone of Fifty / Fifty example app with PHP SDK

## Why?

Example app with usage of new PHP SDK (currently 5.0.0-beta.0)

https://github.com/pipedrive/client-php/blob/5.0.0-beta.0/README.md

## How to test?

Read instructions in README and check demo recording, or follow the next steps:

Required: PHP >= 8.1 (recommended) with sqlite.

```
- run `composer install`
- create a new Marketplace app or use existing one
- copy .env.example to .env and update it with your app properties:
  - PIPEDRIVE_CLIENT_ID=
  - PIPEDRIVE_CLIENT_SECRET=
  - PIPEDRIVE_REDIRECT_URI=http://127.0.0.1:8000/auth/pipedrive/callback
- run `npm run dev` to watch FE assets
- run `php artisan key:generate`
- run `php artisan serve`
```

*NB*: Make sure your app has `deals:full` scope to update deals.

### Expected result

![demo](https://github.com/pipedrive/example-apps/assets/333910/58f38e40-dbc4-47b2-96d5-bb498842cb2f)
